build/build.go

```diff
@@ -1560,8 +1560,22 @@ func waitContextDeps(ctx context.Context, index int, results *waitmap.Map, so *c
 	return nil
 }
 
-func notSupported(d driver.Driver, f driver.Feature) error {
-	return errors.Errorf("%s feature is currently not supported for %s driver. Please switch to a different driver (eg. \"docker buildx create --use\")", f, d.Factory().Name())
+func cacheExportNotSupported(d driver.Driver) error {
+	return errors.Errorf(`Cache export is not supported for the %s driver.
+Switch to a different driver and try again.
+Learn more at https://docs.docker.com/go/build-cache-backends/`, d.Factory().Name())
+}
+
+func exporterNotSupported(d driver.Driver, f driver.Feature) error {
+	return errors.Errorf(`%s is not supported for the %s driver.
+Switch to a different driver and try again.
+Learn more at https://docs.docker.com/go/build-exporters/`, f, d.Factory().Name())
+}
+
+func multiPlatformBuildNotSupported(d driver.Driver) error {
+	return errors.Errorf(`Multi-platform build is not supported for the %s driver without the containerd image store.
```

Comment:
The other cases should also work with containerd image store if we want to mention that.